clients/src/main/java/org/apache/kafka/common/protocol/Protocol.java

```diff
@@ -519,20 +538,59 @@
                                                                        new ArrayOf(FETCH_REQUEST_TOPIC_V0),
                                                                        "Topics to fetch in the order provided."));
 
+    // FETCH_REQUEST_V4 added a per-partition log_start_offset field - the earliest available offset of partition data that can be consumed.
+    public static final Schema FETCH_REQUEST_V4 = new Schema(new Field("replica_id",
```

Comment:
I have updated `upgrade.html` to change latest version from 0.10.3.0 to 0.11.0 and specify change in the FetchRequet and FetchResponse.